tracing/tracing.go

```diff
@@ -1,24 +1,26 @@
 package tracing
 
 import (
-	"fmt"
 	"io"
-	"io/ioutil"
-	"os"
 
+	"github.com/pkg/errors"
 	jaegercfg "github.com/uber/jaeger-client-go/config"
 	jaegerprom "github.com/uber/jaeger-lib/metrics/prometheus"
 )
 
+// ErrInvalidConfiguration is an error to notify client to provide valid trace report agent or config server
+var (
+	ErrInvalidTraceConfiguration = errors.New("no trace report agent or config server specified")
```

Comment:
I don't see this as "Invalid"; it was previously a legitemate way to disable tracing.
How about "Blank"?

Reply:
Yeah `Blank` makes sense. Have made the change.